pom.xml

```diff
@@ -38,6 +38,12 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+     <!-- Google Cloud Storage -->
```

Comment:
@SantiM1 funny thing is I add the google cloud storage dependency to my pom.xml too we might run into a merge conflict, if we do I can fix it no worries.

Reply:
Agree